config/block.block.mainnavigation.yml

```diff
@@ -1,16 +1,14 @@
-uuid: 4b66f029-b2ec-47e5-b470-2418a463d3f4
```

Comment:
Why the file is renamed?

Reply:
As I added the main menu to the blog page it was created.

Reply:
As I had removed the Main Navigation from the Primary Menu region in Block Layout structure, it was removed from the default menu of Bartik theme. Hence the file was deleted. 
Now I added the Main Navigation menu to Block Layout structure, it was not set as default in the Bartik theme. So the file name was changed.